src/Services/PaymentInfo.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Ebanx\Benjamin\Services;
+
+use Ebanx\Benjamin\Models\Configs\Config;
+use Ebanx\Benjamin\Services\Adapters\PaymentInfoAdapter;
+use Ebanx\Benjamin\Services\Http\Client;
+
+class PaymentInfo
+{
+    /**
+     * @var Config
+     */
+    private $config;
+
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    public function __construct(Config $config)
+    {
+        $this->config = $config;
+        $this->client = $this->client ?: new Client();
+    }
+
+    /**
+     * @param string $hash
+     * @param boolean   $isSandbox
+     * @return array
+     */
+    public function getPaymentInfoByHash($hash, $isSandbox = null)
```

Comment:
Why getPaymentInfoByHash if the class is PaymentInfo. Maybe `getByHash` is good.